sns_ik_examples/README.md

```diff
@@ -0,0 +1,54 @@
+# README:  sns_ik_examples
+
+This directory contains scripts that run some tests of the SNS-IK library using
+the Sawyer and Baxter robot models.
+
+## Tutorial: how to run these scripts
+
+First you need to setup your catkin workspace. If you are unfamiliar with this
+process then you should refer to the ROS tutorial page:
+http://wiki.ros.org/catkin/Tutorials/create_a_workspace
+
+Once you set up the catkin workspace it should look something like:
+````
+ros_ws/
+- src/
+  - sns_ik/
+  - baxter_robot/
```

Comment:
erm, it's actually baxter_common/

Reply:
So why did we merge this before changing this? Guess we can change this in a future commit.

Reply:
Your second sentence :)